test/bdd/dockerutil/compose.go

```diff
@@ -152,7 +152,7 @@ func (c *Composition) GenerateLogs(dir, logName string) error {
 		return err
 	}
 
-	f, err := os.OpenFile(logName, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0600)
+	f, err := os.OpenFile(logName, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0600) //nolint:gosec
```

Comment:
@fqutishat @rolsonquadras Does this warrant a followup story?

(we should be careful about disable the security check)

Reply:
Bdd test not important

Reply:
Your comment is correct if it's part of the code

Reply:
Ah, makes sense. I didn't realize it was in the bdd test.